react-backend/routes/cbauthentication/registerCB.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+
+const router = express.Router();
+
+const hashCB = require('../../functions/cbhash');
+const putCBData = require('../../database/queries/CBqueries/putCBData');
+const sendCBemail = require('../../functions/sendCBemail');
+
+let details = {};
```

Comment:
why the global?
does it not work if you set `const details = JSON.parse(body)` on line 21?